### DISCUSSION TITLE
Improvement

### DISCUSSION THREAD

**Discussion by kigwa-martha** [oppia/oppia-android#4919] [General Discussion]
Is there a way the app could be modified in such a way that when a student takes a break can come back and continue from where they left? Meanwhile, the student has to start all over again. This is tiresome to most especially those who cannot navigate on their own. 
Kindly check on that. Thank you.

**Comment by BenHenning** [comment]
Hi @kigwa-martha. Thanks for filing this issue!

The app currently records partial progress for lessons that haven't yet been fully completed, and resumes exactly where the learner left off. This is the intended & tested behavior--are you seeing something different when using the app? If so, could you please detail the steps that you're doing that's leading to losing progress mid-lesson? That may indicate an issue with the app that'd we be really interested in investigating.

**Reply by kigwa-martha** [reply]
Issue sorted. Thanks
